src/main/java/com/alibou/security/config/JwtAuthenticationFilter.java

```diff
@@ -36,17 +37,19 @@ protected void doFilterInternal(
       @NonNull HttpServletResponse response,
       @NonNull FilterChain filterChain
   ) throws ServletException, IOException {
-    if (request.getServletPath().contains("/api/v1/auth")) {
-      filterChain.doFilter(request, response);
-      return;
-    }
     final String authHeader = request.getHeader("Authorization");
     final String jwt;
     final String userEmail;
-    if (authHeader == null ||!authHeader.startsWith("Bearer ")) {
+
+    if(
+        Arrays.asList(SecurityConfiguration.whiteListedRoutes).contains(request.getServletPath()) ||
```

Comment:
change `Arrays.asList(SecurityConfiguration.whiteListedRoutes).contains(request.getServletPath())`
to `Arrays.stream(SecurityConfiguration.whiteListedRoutes).anyMatch(route -> antPathMatcher.match(route, req.getServletPath()))`